### ISSUE TITLE
Change the default book cover image 

### ISSUE THREAD

**Issue by LeighKP** [pressbooks/ideas#365]
### Feature Description

Please switch the default book cover (the red type writer and the word Pressbooks) to an updated version that does not feature a type writer. This is a bandaid solution until we can do something more dynamic (e.g. a cover generator, Cesar once made one for Rebus that was super cool). 

### Feature Use Case

Now that the search results display sorted by most recently updated, a lot more red type writers are showing up. The red type writers conflict with our innovative brand and make the search results look less impressive than they are.

### Other Notes

One option for the cover would be to have the same graphic designer [Leah Concepcion-Vanderbyl](https://www.leahcv.com/) who did the website and some of the assets do a standard book cover. I would be happy to facilitate the conversation.

**Comment by SteelWagstaff** [comment]
Related to larger epic: https://github.com/pressbooks/pressbooks/issues/2097

**Comment by SteelWagstaff** [comment]
Hugh's requests: 
1. good background colour contrast against black/grey background (colour palette drawn from within pressbooks colour family)
2. capture book & digital with our illustrations (not book and book)
3. show how new cover looks on webbook home page & directory

**Comment by SteelWagstaff** [comment]
The default cover image (and user uploaded cover image) is displayed on the webbook's homepage and included as an internal cover image for ebook exports.

Here are some recommended cover image specs for ebooks:

Amazon: For cover images included with books: "Kindle eBooks must have a marketing cover image provided for use on the website detail page. This is provided separately from the eBook file. The ideal size of your eBook cover art is a height/width ratio of 1.6:1. ... the height of the image should be at least 2,500 pixels. Ideal dimensions for cover files are 2,560 x 1,600 pixels."

Internal cover images: "Kindle eBooks must have an internal cover image provided for use within the book content. Provide a large, high-resolution cover, because Amazon quality assurance will fail the book if the cover is too small. ... Note: This cover image guidance doesn't apply to KDP eBooks. If you're using KDP, follow the KDP guidelines for cover images."

https://kdp.amazon.com/en_US/help/topic/G6GTK3T3NUHKLEFX

Apple: For cover images included with books: "The book’s cover art must use RGB color mode and should be at least 1400 pixels along the shorter axis. For best results, a good rule of thumb is to use an image that is a minimum of 300 dpi." 

For interior cover images: "Apple recommends that you set the colorspace on your book images (for both EPUB and Multi-Touch books) to sRGB. ... Images that have any transparent areas should be PNG format or, ideally, use JPEGs with WebKit PNG masks. Images that do not have transparency should be JPEGs. ... Images (within the EPUB) cannot exceed 4 million pixels. Apple recommends providing images that are at least 1.5 times the intended viewing size up to a maximum of 4 million pixels. ... In Fixed Layout books, publishers can scale images down using CSS. For example, an image that is defined as 40px wide in CSS should actually be 60px wide. A full-bleed image should be no smaller than 738 x 985" https://help.apple.com/itc/booksassetguide/#/itca71ad3c33

Ingram Spark: for cover images included with books: "File must contain front cover only. Full spread print book jackets that include spine and back cover will be rejected. Must be a JPG File in RGB. 2560 pixels on longest side. Minimum 1600 pixels on shortest side."

for interior cover images: "Include an internal cover image. This should be formatted the same size of, and as part of your interior. (For use within the book content.)"

**Comment by colomet** [comment]
I think is important to decided the aspect ratio and if Amazon have the most limited dimensions 2,560 x 1,600 pixels, we must keep that consistency, in order to be able to work with them, since by working with Amazon, we are able to work with apple too, and maybe many other platforms, since they would use to have those main platforms as reference.

And to maintain our site different than theirs, would mind a duplication effort.

**Comment by SteelWagstaff** [comment]
Hi @colomet, just to clarify, the cover image uploaded to Pressbooks via book info isn't ever used or consumed by Amazon or other book publishing services. This cover image is only displayed on the book home page and correspondes to the 'interior cover image' described in the passages in original ticket. Cover images uploaded to Amazon and other services have to be uploaded separately.

**Comment by colomet** [comment]
yes, I know, but since we can have the same image ratio and size. Why not to upload our image with the amazon size in order to not to pay twice to an illustrator?

In that way, if in the future we do upload the book to amazon, we can use the same image.


i think, notting chages to pressbooks and helps authors in the future.
